mlx_vlm/tokenizer_utils.py

```diff
@@ -314,7 +314,8 @@ def load_tokenizer(model_path, return_tokenizer=True, tokenizer_config_extra={})
 
     tokenizer_file = model_path / "tokenizer.json"
     if tokenizer_file.exists():
-        tokenizer_content = json.load(tokenizer_file.open())
+        with open(tokenizer_file, "r") as f:
+            tokenizer_content = json.load(f)
```

Comment:
Nit:
We need to handle errors when loading. 

So lets import`from json import JSONDecodeError` and do a try catch.

```suggestion
        with open(tokenizer_file, "r") as f:
            try:
                tokenizer_content = json.load(f)
            except JSONDecodeError as e:
                raise JSONDecodeError(
                    "Failed to parse tokenizer.json", 
                    e.doc, 
                    e.pos
                )
```